### DISCUSSION TITLE
Project disappears after creation

### DISCUSSION THREAD

**Discussion by bmwatson214** [GoogleChrome/lighthouse-ci#665] [Q&A] [answered]
**Describe the bug**
After creating a project using `lhci wizard` I see it in my UI at first and then minutes later it is no longer accessible.
I get "Oops! Unable to find the specified resource." when trying to navigate to the project.

**To Reproduce**
1. Create docker image (see [gist code](https://gist.github.com/bmwatson214/14e9fdbe0eaaea363c5cbe3e58b57aee))
2. Push image to AWS ECR repo (using terraform - this [gist has summarized configuration](https://gist.github.com/bmwatson214/375981e79b86443e8bd2e3eded598fdc))
3. Use AWS ECS to run the container (accessible via load balancer)
4. Locally use lhci wizard to setup the project ([gist with basic flow](https://gist.github.com/bmwatson214/74aac7326e5ffbf7aaf4e77fc02d4b67))
5. Navigate to server ui and see project exists
6. Refresh 10-15 minutes later and project no longer exists [BUG - or possibly I have misconfigured]

**Expected behavior**
I can't figure out why my project isn't persisting? Am I not using the volumes correctly? It seems that maybe when new tasks are spun up by the service my data is lost?

**Environment (please complete the following information):**
 - OS: mac
 - Browser: chrome
- Other: Node container

Thanks in advance for your help!

**Comment by patrickhulce** [comment, marked as answer]
The docker image example you've provided is using sqlite as the data storage mechanism but isn't persisting the location of that db anywhere. Every time your image starts you're starting over with a fresh database.

We've provided an example of how to accomplish this but specific debugging of individual setups is beyond the scope of what we can help with. Someone else might want to chime in with AWS help :)

**Reply by bmwatson214** [reply]
Ahh yes, that makes sense! Thank you for clearing that up for me! We are planning on building out the db next :)
